docs/5.0/pages/enterprise/ssh-kubernetes-fedramp.md

```diff
@@ -20,8 +20,8 @@ government agencies.
 | [AC-03 Access Enforcement](https://nvd.nist.gov/800-53/Rev4/control/AC-3)   | Teleport Enterprise supports robust [Role-based Access Controls (RBAC)](./ssh-rbac.md) to: <br />• Control which SSH nodes a user can or cannot access. <br />• Control cluster level configuration (session recording, configuration, etc.) <br />• Control which UNIX logins a user is allowed to use when logging into a server. |
 | [AC-10 Concurrent Session Control](https://nvd.nist.gov/800-53/Rev4/control/AC-10)   | Teleport administrators can define concurrent session limits using Teleport’s RBAC. |
 | [AC-17 Remote Access](https://nvd.nist.gov/800-53/Rev4/control/AC-17)   | Teleport administrators create users with configurable roles that can be used to allow or deny access to system resources. |
-| [AC-20 Use of External Information Systems](https://nvd.nist.gov/800-53/Rev4/control/AC-20)  | Teleport supports connecting multiple independent clusters using a feature called [Trusted Clusters](../../trustedclusters/). When allowing access from one cluster to another, roles are mapped according to a pre-defined relationship of the scope of access.|
-| [AU-03 Audit and Accountability](https://nvd.nist.gov/800-53/Rev4/control/AU-3) – Content of Audit Records and [AU-12 Audit Generation](https://nvd.nist.gov/800-53/Rev4/control/AU-12) | Teleport contains an [Audit Log](../../architecture/authentication/#audit-log) that records cluster-wide events such as: <br />• Failed login attempts.<br />• Commands that were executed (SSH “exec” commands).<br /> • Ports that were forwarded. <br />• File transfers that were initiated.|
+| [AC-20 Use of External Information Systems](https://nvd.nist.gov/800-53/Rev4/control/AC-20)  | Teleport supports connecting multiple independent clusters using a feature called [Trusted Clusters](../trustedclusters/). When allowing access from one cluster to another, roles are mapped according to a pre-defined relationship of the scope of access.|
+| [AU-03 Audit and Accountability](https://nvd.nist.gov/800-53/Rev4/control/AU-3) – Content of Audit Records and [AU-12 Audit Generation](https://nvd.nist.gov/800-53/Rev4/control/AU-12) | Teleport contains an [Audit Log](../architecture/authentication/#audit-log) that records cluster-wide events such as: <br />• Failed login attempts.<br />• Commands that were executed (SSH “exec” commands).<br /> • Ports that were forwarded. <br />• File transfers that were initiated.|
```

Comment:
These link didn't work according to the latest semrush report, so I fixed them. Not sure how they worked before.